skpro/distributions/base/_base.py

```diff
@@ -529,6 +529,10 @@ def pdf(self, x):
         ``pd.DataFrame`` with same columns and index as ``self``
             containing :math:`p_{X_{ij}}(x_{ij})`, as above
         """
+        distr_type = self.get_tag("distr:measuretype", "mixed", raise_error=False)
```

Comment:
I am not really sure about this, but is it okay if we assume that the distribution is mixed (continuous and discrete)?

Reply:
that happens only if the tag is not set.

Since the "exceptional" behaviour happens for continuous/discrete ("return 0" etc), this choice ensures that no exceptional behaviour happens, and we try to call the inner `_pdf` etc.

So, the purpose is not assuming that the distribution is mixed, but just for the purpose of calling the inner methods in all pdf/pmf related functions, since there is not enough information.